ClientApp/viking-vault/src/display_user_list/AdminPage.tsx

```diff
@@ -84,6 +84,16 @@ class AdminPage extends React.Component<any, IProfileData>{
         }
     }
 
+    private deleteUserFromComponent = (email : string) =>{
```

Comment:
shouldn`t this method remove a user from a list or smth? why are you filtering that list instead of deleting the user?

Reply:
I noticed that using filter instead is a clean and elegant way of deleting a user from the list. There are many places where people advise using filter for deleting an entry from a list. Also, I didn't notice any explicit remove or delete functions. Another popular approach is to retrieve the index of the element and then split the list, but I think that using filter is neater.